source/meat/honeyMustardPorkTenderloin.rst

```diff
@@ -0,0 +1,33 @@
+Honey Dijon Garlic Roasted Pork Tenderloin 
+==========================================
+
+Details
+-------
+
+* Number of servings: 4 
+* Cook time: 45 minutes
+
+Ingredients
+-----------
+
+* 2 lbs pork tenderloin
+* 1/4 cup whole grain dijon mustard
+* 2 tablespoons honey
+* 4 cloves garlic crushed
+* 1/2 teaspoon salt
+* 1/4 teaspoon black pepper
+
+Steps
+-----
+
+#. Preheat oven to 400.
```

Comment:
Have we been using units for things like this?

Reply:
Previously we have not. I removed the F in the last step to stick with this convention. If we want to make it more globally usable, might be worth considering adding in the future.